src/metamorpho/MetamorphoSnippets.sol

```diff
@@ -102,13 +110,13 @@ contract MetamorphoSnippets {
         // Get the supply rate
         uint256 utilization = totalBorrowAssets == 0 ? 0 : totalBorrowAssets.wDivUp(totalSupplyAssets);
 
-        supplyRate = borrowRate.wMulDown(1 ether - market.fee).wMulDown(utilization);
+        supplyRate = borrowRate.wTaylorCompounded(1).wMulDown(1 ether - market.fee).wMulDown(utilization);
```

Comment:
why do we divide by the utilization?

Reply:
Because that's how the supply rate is computed from the borrow rate right ?